### PR TITLE
Update permission network style, content polish

### PR DESCRIPTION
**Pull request by NukeManDan**
- Add detail and snipped code examples
- 100 chars or less per line
- removed deprecated link
- use `make` build flow

### CODE REVIEW

**Comment by NukeManDan** [comment]
needs `prettier` run before merge

File: docs/tutorials/build-permission-network/add-node-authorization-pallet.md [hunk 1]
```diff
@@ -2,49 +2,99 @@
...
+With this pallet, you have two ways to authorize a node which wants to join the network:
+
+1. Join the set of well known nodes between which the connections are allowed.
+    Such effort usually needs to be approved by the governance in the system.
+2. Ask for the connection from a node which is already a member of the network.
+    Such node can either be a well known node or a normal one. 
+
+A node associated with a `PeerId` must have **one and only one owner**.
+The owner of a well known node is specified when adding it.
+If it's a normal node, *any* user can claim a `PeerId` as its owner.
```
Comment (kaichaosun) [inline docs/tutorials/build-permission-network/add-node-authorization-pallet.md:21]:
```suggestion
If it's a normal node, *any* user can claim the `PeerId` as its owner.
```

File: docs/tutorials/build-permission-network/add-node-authorization-pallet.md [hunk 2]
```diff
@@ -2,49 +2,99 @@
...
+
+It uses [offchain worker](https://substrate.dev/docs/en/knowledgebase/runtime/off-chain-workers)
+to set authorized nodes in node-authorization pallet. Make sure to enable offchain worker with
+the right CLI flag as offchain worker is diabled by default for non-authority nodes.
+Your node can be lagged with the latest block, in this case you need to disable offchain worker
+and manually set reachable reserved nodes to sync up with the network.
+
+## Add the `node-authorization` pallet
+
+If you already hace Node Template cloned, you can just create a
```
Comment (kaichaosun) [inline docs/tutorials/build-permission-network/add-node-authorization-pallet.md:39]:
```suggestion
If you already have Node Template cloned, you can just create a 
```

**Review by kaichaosun — APPROVED** [review]
👍

**Pull request merged**